### PR TITLE
KOGITO-9053: [Diagram Viewer] stateDataFilter doesn't show details in diagram properly

### PR DESCRIPTION
**Pull request by treblereel**
JIRA: https://issues.redhat.com/browse/KOGITO-9053

### CODE REVIEW

**Review by romartin — COMMENTED** [review]
Does it makes sense to you @treblereel to add some unit test for this regexp logic only?

**Review by LuboTerifaj — CHANGES_REQUESTED** [review]
Hello @treblereel ,

the PR is missing tests. Could  you please add some?

Thanks!

**Comment by treblereel** [comment]
@romartin I planned to add tests to all yaml  examples I saw ( well, yaml from test.html, vscode tests and Lubos examples) as separate PR. But if we want to have a test for this regex fix, yeap, why not, it's easy.

**Review by romartin — APPROVED** [review]
great, thanks @treblereel

File: packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-kogito-app/src/test/resources/org/kie/workbench/common/stunner/sw/client/selenium/yaml/FoodOrderWorkflow.sw.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - functionRef:
+          refName: Charge For Order Function
+          arguments:
+            order: "${ .order.id }"
+        actionDataFilter:
+          results: "${ .outcome.status }"
+          toStateData: "${ .status }"
+    end: true
+    stateDataFilter:
+      output: '.status} | .orderstatus += ["Order Completed"] }'
```
Comment (LuboTerifaj) [inline packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-kogito-app/src/test/resources/org/kie/workbench/common/stunner/sw/client/selenium/yaml/FoodOrderWorkflow.sw.yaml:38]: It looks like the output value is incomplete. Could you please update it?

**Review by LuboTerifaj — CHANGES_REQUESTED** [review]
Hello @treblereel ,

I have an inline comment regarding test.

There is also an issue with opening [finalize-college-application.sw.yaml](https://github.com/LuboTerifaj/specification/blob/examples-in-separate-files/examples/resources/finalize-college-application.sw.yaml) workflow. Loading takes very long time in comparison with main. Could you please have a look at it?

Thanks!

**Comment by treblereel** [comment]
@LuboTerifaj let's create separate a Jira task for the performance issue. And which browser is slow?

**Comment by LuboTerifaj** [comment]
Hello @treblereel,

 I tested it using VSCode extension. I think we should not introduce such regression. Anyway, I reported the jira in case we decide to fix it separately:
- [KOGITO-9222](https://issues.redhat.com/browse/KOGITO-9222) _SWF - Workflow yaml file takes too much time to load_.

cc @romartin 

Could you please update the suggested change from my previous comment?

Thanks!

**Comment by romartin** [comment]
Thanks @LuboTerifaj  for the feedback.

We have some important release in a few weeks, so I think we should address also https://issues.redhat.com/browse/KOGITO-9222 as part of this work, otherwise, if merging it implies the issues, we should wait for the release.

So just let us know @treblereel if it is related, also if it is feasible to fix, before second week of June.

Thanks!

**Comment by LuboTerifaj** [comment]
The issue will be fixed in https://github.com/kiegroup/kie-tools/pull/1680. @treblereel , please close this pull-request. Thanks!

**Pull request closed without merging**